src/ui/autocomplete.ts

```diff
@@ -275,6 +283,8 @@ class AutocompleteUI {
     results.forEach((result, index) => {
       const li = document.createElement('li');
       li.classList.add(CLASSNAMES.RESULTS_ITEM);
+      li.setAttribute('role', 'option');
+      li.setAttribute('id', `item-${index}`);
```

Comment:
fyi we use the same `class` value for both `class` and `id` attributes for the `input` element but different values for this `li`element. Should this be 
```suggestion
      li.setAttribute('id', `${CLASSNAMES.RESULTS_ITEM}-${index}`);
```
to make it more consistent?

Reply:
we should prob make this `${CLASSNAMES.RESULTS_ITEM}-${index}` since `id` attributes should be unique. `item-${index}` might cause issues with existing ids